tools/ngsderive.wdl

```diff
@@ -69,10 +69,10 @@ task strandedness {
             "$CWD_BAM" \
             > ~{outfile_name}
 
-        if split_by_rg; then
-            awk 'NR > 1' ~{outfile_name} | grep 'overall' | cut -f6 > strandedness.txt # TODO broken until ngsderive v4 is released
+        if ~{split_by_rg}; then
+            echo "N/A" > strandedness.txt
```

Comment:
Does `ngs strandedness` not output an overall call when running in per-RG mode?

Reply:
Good question! It did not, until I released v4. As you'll see, in WDL we're stuck on v3. This is because there is an issue with the conda build preventing v4 from being released (and by extension, it isn't released on BioContainers either).

I've spent a decent amount of time trying to get to the bottom of the conda build error without luck. I pinged the conda team asking them to look at it, and someone _did_ take a look. They just didn't offer any helpful advice and I gave up TBH. And that was quite a long time ago now.

TBH I expected I could cut the Rust port of `ngsderive` sooner than has materialized. IMO the v4 release isn't important enough to deal with that headache **in light of the Rust port releasing soon.** The PR is in Clay's hands for review. The plan is to archive the entire `ngsderive` repo once that port is merged. So no sense in spending time on addressing that when it's on the verge of archival.